src/main/java/org/spongepowered/api/world/ChunkTicketManager.java

```diff
@@ -204,6 +204,14 @@
          * @return The companion data in the form of a data container
          */
         DataContainer getCompanionData();
+        /**
+         * Gets the world passed when creating this ticket
+         *
+         * @return The World object given to this ticket upon its creation
+         */
+        World getWorld();
+
+        // TODO: NBTTag getCustomData(); (Is saved with ticket information)
```

Comment:
We added this before getCompanionData was added. On a side note, getWorld is part of `Ticket`, the Forge class this feature wraps. It might not be necessary anymore :/ but I was mostly just clearing the merge conflict.